desktop/apps/artwork/components/auction/view.coffee

```diff
@@ -7,7 +7,7 @@ openBuyersPremiumModal = require './components/buyers_premium/index.coffee'
 AuthModalView = require '../../../../components/auth_modal/view.coffee'
 inquire = require '../../lib/inquire.coffee'
 acquire = require '../../lib/acquire.coffee'
-helpers = require './helpers.coffee'
+ViewHelpers = require './helpers.coffee'
```

Comment:
Why change the name? `helpers` as a var maps directly to the filename, which makes things easier to track down and refactor later